README.md

```diff
@@ -22,6 +22,17 @@ The recommended way to develop docs.rs is a combination of `cargo run` for
 the main binary and [docker-compose](https://docs.docker.com/compose/) for the external services.
 This gives you reasonable incremental build times without having to add new users and packages to your host machine.
 
+### Git Hooks
+
+For ease of use, `git_hooks` directory contains useful `git hooks` to make your development easier.
+
+```bash
+# Unix
+cd .git/hooks && ln -s ../../.git_hooks/* . && cd ../..
```

Comment:
Does this need to be a whole directory? It's only one file.

Also the command can be simplified:
```suggestion
ln -s ../../.git_hooks/* .git/hooks
```

Reply:
`zsh: no matches found: ../../.git_hooks/*` you need the `cd` without it it won't find the source file.

Since the `README.md` is assumes you're in the root (also `.git/hooks` path assumes that too).

Also if you try `ln -s .git_hooks/* .git/hooks` the link created isn't correct, the combo with `cd` seems to be the most robust way I found.

Reply:
Regarding the one file, I assume more will be added, `pre-push` for `cargo test` for example as discussed in the previous comment.

Reply:
Oh I see, because you're using shell expansion here. Yeah this is fine then.